### PR TITLE
Add Eaternity

### PR DESCRIPTION
**Pull request by mklarmann**
- [x ] I read [CONTRIBUTING.md](https://github.com/pogopaule/awesome-sustainability-jobs/blob/main/CONTRIBUTING.md)  
- [x ] I didn't change README.md (but instead src/data.yaml or src/template.md)

### CODE REVIEW

File: src/data.yaml [hunk 1]
```diff
@@ -1389,6 +1389,19 @@ jobs:
         long: 13.4877535
     review:
     rating:
+  - name: eaternity
+    website: https://eaternity.org
+    jobs: https://eaternity.org/about/#job-offer
+    description: CO2 calculation for food.
+    remote: false
+    speculative: false
+    field: Carbon Analysis of Foods
```
Comment (pogopaule) [inline src/data.yaml:1398]: There is just this general category

```suggestion
    field: Carbon Analysis, Capture and Offsetting
```

**Review by pogopaule — CHANGES_REQUESTED** [review]
Thank you for your contribution! There is just this minor change you need to make.

**Comment by mklarmann** [comment]
ok

**Comment by pogopaule** [comment]
Did you really want to close the PR? It looks good to me.

**Pull request closed without merging**